packages/image_picker/image_picker_ios/ios/Classes/FLTImagePickerPlugin.m

```diff
@@ -129,14 +134,24 @@ - (void)launchUIImagePickerWithSource:(nonnull FLTSourceSpecification *)source
   imagePickerController.delegate = self;
   imagePickerController.mediaTypes = @[ (NSString *)kUTTypeImage ];
   self.callContext = context;
+  BOOL requestFullMetadata = context.requestFullMetadata;
```

Comment:
Same here. Especially since this is only used once, and conditionally, so doesn't make sense to define at such a high level.